src/@next/Bar/BarStyle.ts

```diff
@@ -2,14 +2,20 @@ import styled from 'styled-components';
 import * as Breakpoints from '../utilities/breakpoints';
 import { Neutral } from '../utilities/colors';
 import { space8 } from '../utilities/spacing';
+import React from 'react';
 
-export const StyledBar = styled.div`
+interface StyledBarProp extends React.HTMLAttributes<HTMLDivElement> {
+  smallScreen: boolean;
+}
+
+export const StyledBar = styled.div<StyledBarProp>`
   display: flex;
   flex-direction: row;
-  padding: 20px 0;
+  padding: ${props => (props.smallScreen ? '8' : '12')}px 0;
   width: 100%;
   position: absolute;
-  height: 90px;
+  min-height: ${props => (props.smallScreen ? '70' : '90')}px;
+  height: fit-content;
 
   @media (max-width: ${Breakpoints.large}) {
```

Comment:
@michac789 Given that we are using min-height here, I believe we should remove the height attribute to prevent content overflow.